fjord/analytics/analyzer_views.py

```diff
@@ -223,27 +220,32 @@ def analytics_search(request):
     search_date_end = smart_date(
         request.GET.get('date_end', None), fallback=None)
     search_bigram = request.GET.get('bigram', None)
+    selected = request.GET.get('selected', None)
+
+    search_has_email = request.GET.get('has_email', None)
+    search_country = request.GET.get('country', None)
+    search_domain = request.GET.get('domain', None)
+    search_api = smart_int(request.GET.get('api', None), fallback=None)
```

Comment:
I moved these down here because this view is almost a copy of the non-analyzer frontpage dashboard. In moving these to this section, it makes it easier to compare the dashboard view with this view to see differences.